### PR TITLE
Add Danish holidays etc.

### PR DESCRIPTION
**Pull request by theAsylum-dk**
UTF-8 symbols represent Flag days (🏴), bank holidays(🚪), and contractual holidays (🏲)

### CODE REVIEW

**Comment by lsalvadore** [comment]
I am sorry, watching the repository was disabled and I only saw your pull request now. Now I am watching the repository, so this should not happen again.

I will review your pull request as soon as possible.

**Comment by theAsylum-dk** [comment]
No worries - a few days/weeks more or less probably won't cause any issues ;)

**Review by lsalvadore — CHANGES_REQUESTED** [review]
Syntax looks good and about the Danish content I am going to simply trust you.
Only thing that needs to be addressed is the directory name: as you can see from the other languages, da_DK.UTF8 is expected instead of da_DK.

Please also note that before merging this request I would like to solve an issue about license: the license was forgotten when the repository was created. I have created pull request #9 to fix the issue. Please keep in mind that this request will be merged with the license that will be added through that pull request, so if for any reason you disagree with it, please warn me before I merge this pull request. Also please inform me if you disagree with that license and you contributed in the past to the repository, so that we can remove your contribution or license it differently.
You might also like to help reviewing the pull request itself.

Thanks.

**Review by theAsylum-dk — COMMENTED** [review]
I hope this conforms a little better? :)

Also: I have no issues with the licensing - please do as you see fit.

**Comment by theAsylum-dk** [comment]
I hope this latest commit conforms a little better? :)

Also: I have no issues with the licensing - please do as you see fit.

File: calendar.danish [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/*
+ * Danish calendar file(s)
+ *
+ */
+
+#ifndef _calendar_danish_
+#define _calendar_danish_
+
+#include <da_DK/calendar.all>
```
Comment (lsalvadore) [inline calendar.danish]: I am sorry, there is still one reference to da_DK directory that needs to be switched to da_DK.UTF-8. Once this is fixed too, I think it is ready to be merged.

Comment (theAsylum-dk) [reply calendar.danish]: Ooops - I'm sorry for the oversight. This is now fixed.

**Review by lsalvadore — APPROVED** [review]

**Comment by lsalvadore** [comment]
Merged, thanks for your contribution.

**Pull request merged**